tests/drivers/dma/chan_blen_transfer/boards/adp_xc7k_ae350.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_ICACHE=n
+CONFIG_DCACHE=n
```

Comment:
Again, this is odd. How the cache is managed on this platform?

Reply:
It's because there is no io coherent in adp_xc7k_ae350.
So if the dcahce is enable, the data in cache may not different with memory.

But the icache can be openend exactly, thanks for your remind.